tasks/src/main/scala/dagr/tasks/fgbio/FilterConsensusReads.scala

```diff
@@ -29,26 +29,38 @@ import dagr.tasks.DagrDef.PathToBam
 
 import scala.collection.mutable.ListBuffer
 
-class FilterConsensusReads(val in: PathToBam,
-                           val out: PathToBam,
-                           val ref: PathToFasta,
-                           val reverseTags: Boolean = false,
-                           val minReads: Int,
-                           val maxReadErrorRate: Double,
-                           val minQuality: Int,
-                           val maxBaseErrorRate: Double,
-                           val maxNoCallFraction: Double)
+object FilterConsensusReads {
+  def apply(in: PathToBam,
+            out: PathToBam,
+            ref: PathToFasta,
+            minReads: Int,
+            maxReadErrorRate: Double,
+            maxBaseErrorRate: Double,
+            minQuality: Int,
+            maxNoCallFraction: Double): FilterConsensusReads = {
+    apply(in=in, out=out, ref=ref, minReads=Seq(minReads), maxReadErrorRate=Seq(maxReadErrorRate),
+      maxBaseErrorRate=Seq(maxBaseErrorRate), minQuality=minQuality, maxNoCallFraction=maxNoCallFraction)
+  }
+}
+
+case class FilterConsensusReads(in: PathToBam,
+                                out: PathToBam,
+                                ref: PathToFasta,
+                                minReads: Seq[Int],
+                                maxReadErrorRate: Seq[Double],
+                                maxBaseErrorRate: Seq[Double],
```

Comment:
The `fgbio` tool requires at least one value for each of these arguments.  I.e. to turn off the filtering you set it to `1.0`, but you don't omit it.  So defaulting to `Seq.empty` here would be misleading I think.